openshift/templates/web/web-deploy.dev.param

```diff
@@ -23,8 +24,8 @@ APPLICATION_DOMAIN=web-jag-shuber-dev.pathfinder.gov.bc.ca
 # SITEMINDER_WHITE_LIST=
 # ROUTE_TIMEOUT=60s
 TAG_NAME=dev
-# MIN_REPLICAS=1
-# MAX_REPLICAS=4
+# MIN_REPLICAS=3
```

Comment:
what is the motivation for this? or user's seeing a delay when requesting the app?

Reply:
Two motivations, better tolerance to node evacuations, and better application response.